frontend/src/components/navbar/Navbar.js

```diff
@@ -19,18 +20,10 @@ function Navbar() {
 
   useEffect(() => {
     const token = localStorage.getItem("accessToken");
-    const config = {
-      headers: {
-        "Content-Type": "application/json",
-        Authorization: `Bearer ${token}`,
-      },
-    };
-    token &&
-      Axios.get(`${process.env.REACT_APP_BASE_URL}/user`, config)
-        .then((response) => {
-          setCurrentUser({ ...response.data });
-        })
-        .catch((error) => console.log(error));
+    if (token) {
+      isLoggedIn = true;
```

Comment:
In react can we directly assign values to state? Is that right?

Reply:
noi have fixed it